engine/src/mbliphonedialog.mm

```diff
@@ -161,7 +161,7 @@ static void dopopupanswerdialog_prewait(void *p_context)
             for(uint32_t i = 0; i < ctxt -> button_count; i++)
             {
                 UIAlertAction *t_action;
-                t_action = [UIAlertAction actionWithTitle: [NSString stringWithMCStringRef: ctxt -> buttons[i] ]
+                t_action = [UIAlertAction actionWithTitle: MCStringConvertToAutoreleasedNSString(ctxt -> buttons[i) ]
                                                     style: UIAlertActionStyleDefault
```

Comment:
This should be 

`t_action = [UIAlertAction actionWithTitle: MCStringConvertToAutoreleasedNSString(ctxt -> buttons[i]) ]`

(i.e. currently a `]` is missing)